app/views/pages/home.html.erb

```diff
@@ -1,2 +1,3 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+<h2>Welcome to our Awesome Devise Learning Page!</h2>
```

Comment:
Make it into an H1 for SEO reasons

Reply:
Cool, had completely forgotten! Thanks sensei!